modules/local_contacts/modules.php

```diff
@@ -240,7 +240,7 @@ protected function output() {
             $button = '<input type="hidden" name="contact_id" value="'.$this->html_safe($current['id']).'" />'.
                 '<input class="btn btn-primary edit_contact_submit" type="submit" name="edit_contact" value="'.$this->trans('Update').'" />';
         }
-        return '<div class="add_contact kokokoko"><form class="" method="POST">'.
+        return '<div class="add_contact kokokoko"><form class="" method="POST" action="?page=contacts">'.
```

Comment:
what is class `kokokoko` for ?

Reply:
I don't know, it must surely be a mistake slipped in by my predecessor